shark-log/src/main/java/shark/SharkLog.kt

```diff
@@ -13,45 +13,31 @@ object SharkLog {
     /**
      * Logs a debug message formatted with the passed in arguments.
      */
-    fun d(
-      message: String,
-      vararg args: Any?
-    )
+    fun d(message: String)
```

Comment:
I'm thinking we should remove all method that must take a string, it's way too easy to forget to use the callback one and pass in interpolated strings.

Reply:
This one is a part of the interface `Logger`, and since everyone is logging only via `SharkLog.d {}` I felt it would be cleaner to keep real loggers implement this simple interface without lambda.

`SharkLog` only has two methods, both with lambda

Reply:
This one is a part of the interface `Logger`, and since everyone is logging only via `SharkLog.d {}` I felt it would be cleaner to keep real loggers implement this simple interface without lambda.

`SharkLog` only has two methods, both with lambda